commands.go

```diff
@@ -68,6 +68,11 @@ var Commands = []cli.Command{
 				),
 				Value: "none",
 			},
+			cli.StringFlag{
```

Comment:
Instead of parsing them out one by one, we can use the cli package's `StringSliceFlag` which allows you to pass the same option repeatedly and will return an array. Which means you could simplify `hosts.go`'s `parseDockerOpts()` The interface to the user would be: `$ docker-machine create -d <driver> --docker-opt="foo=bar" --docker-opt="baz=qux" ... <name>`